spec/acceptance/hieradata/os/RedHat/7.yaml

```diff
@@ -0,0 +1,2 @@
+---
+redis::globals::scl: rh-redis5
```

Comment:
Should we also switch to PG 12 while we're at it?

Reply:
I'm actually a bit surprised, this still works without it ;)

Reply:
Updated to include this now.